server/lib/renderers/xls.js

```diff
@@ -16,8 +16,16 @@ exports.render = render;
 exports.extension = '.xls';
 exports.headers = headers;
 
-
 async function render(data, template, options) {
   const htmlString = juice(await html.render(data, template, options));
-  return Buffer.from(htmlString, 'utf-8');
+
+  // NOTE(@jniles)
+  // This code replaces the <head></head> contents to avoid CSS <link> errors when
+  // opening in MS Excel.
+  const start = htmlString.substring(0, htmlString.search('<head>'));
+  const end = htmlString.substring(htmlString.search('</head>') + 7);
```

Comment:
Very good, i would like to know what means the `7`, is it possible to use a variable for that.

Reply:
Aha.  7 is the length of the string `'</head>'`.  Since without the 7, it would leave the word `</head>` in the template.

Reply:
Do you have a suggestion for how to do this cleaner?